docker/README.md

```diff
@@ -34,7 +32,7 @@ docker run --cap-add MKNOD --cap-add SYS_ADMIN --device /dev/fuse \
     -e LOCAL_UID="$(id -u)" -e LOCAL_GID="$(id -g)" \
     -v redox-"$(id -u)-$(id -g)"-cargo:/usr/local/cargo \
     -v redox-"$(id -u)-$(id -g)"-rustup:/usr/local/rustup \
-    -v "$(pwd):$(pwd)" -w "$(pwd)" --rm redox make fetch all
+    -v "$(pwd):$(pwd)" -w "$(pwd)" --rm redoxos/redox make fetch all
```

Comment:
If this is to be pushed to dockerhub, I would use another name than `redox` for this image as it's an image of the cross compilation toolchain, not the OS.

Reply:
Hi, thanks for the feedback.

Regarding this comment, @jackpot51 already set it up that way: https://hub.docker.com/r/redoxos/

In my opinion, the naming is okay. Redox is an OS with a non-linux kernel, so the userspace parts could not boot within within a docker container anyways. Hence not so much potential for confusion here.

How would you name it? `redox-toolchain`?

Reply:
Yes, you're right. I guess `redox-toolchain` would be redundant considering the namespace, maybe something like `cross-toolchain`.

Reply:
This is nitpicking, but is it a cross toolchain after all? The docker container is x86_64 and that is also what the target is compiled for.

`redoxos/toolchain` ?

Reply:
I think we can talk about a cross toolchain as it uses a Linux host to build Redox binaries.

But as you pointed out earlier, the fact that it is a docker image makes it obvious that the host is not Redox. Under this perspective, `toolchain` is probably enough for people to understand.